appveyor.yml

```diff
@@ -12,10 +12,17 @@ branches:
   only:
     - master
     - /^release-.*/
+    - /^cv.*/
 
+# Note: use `[ci skip]` or `[skip ci]` in the commit message title and AppVeyor won't be
+# built for that commit. You can use `[skip appveyor]` to explicitly skip AppVeyor and
+# allow other CI to still run.
 skip_commits:
 # Add [av skip] to commit messages for docfixes, etc to reduce load on queue
   message: /\[av skip\]/
+# Skip running CI for changes only to the documentation
+  files:
+    - doc/
```

Comment:
skipping entirely doesn't seem like a good idea, doc-build failures that are platform specific would never get caught until after a merge

Reply:
Thanks for pointing that out. Do you have an example of a platform specific doc-build failure which would be caused by a doc change?

Reply:
I don't have the time available to dig through appveyor logs and find a specific example, but it has happened before. Documentation of platform-specific functionality will need the doc build to get run on each platform. And there have been issues in Documenter's handling of Windows paths that have come up.

Reply:
Ok. If there is a doc-build issue that is specific to Windows I think that is something that Documenter should be testing for. In the event that the Julia documentation triggers a new Documenter issue specific to Windows we can address it at that time. I don't think it's the Julia CI's responsibility to test Documenter.

Reply:
It is the Julia CI's responsibility to make sure changes don't break the build, before the get merged. You'll miss platform-specific instances of that if you go ahead with this change as-is.

Reply:
To play it safe I'll disable this check and try to implement a conditional block like Travis CI in a future PR.